src/containers/Login/Login.tsx

```diff
@@ -39,48 +41,98 @@ export function Login() {
       .catch((err) => console.error(err));
   };
 
-  if (didLoginSucceed) {
-    return <Redirect to="/admin" />;
+  const handlePharmacistSubmit = () => {
+    setIsPharmacist(true);
+    setName("PharmacyUser");
+    setPassword(pharmacyPassword);
+    post({ name, password })
```

Comment:
This line could be `post({ name: "PharmacyUser", password: pharmacyPassword });` ... i think that would clean up the lines above